fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsAccountInterestPostingServiceImpl.java

```diff
@@ -449,7 +449,9 @@ protected void recalculateDailyBalances(final Money openingAccountBalance, final
                 }
 
                 runningBalance = runningBalance.plus(transactionAmount);
-                transaction.updateRunningBalance(runningBalance);
+                if (!transaction.getRunningBalance(transactionAmount.getCurrency()).isEqualTo(transactionAmount)) {
```

Comment:
how can we compare transaction amount with running balance? what is the logic here?

Reply:
Sorry, forgot to remove this thing.